compose.yaml

```
@@ -0,0 +1,20 @@
+name: liconic_module
+services:
+  liconic_module:
+    container_name: liconic_module
+    image: ${IMAGE}
+    build:
+      context: .
+      dockerfile: Dockerfile
+      tags:
+        - ${IMAGE}:latest
+        - ${IMAGE}:${PROJECT_VERSION}
+        - ${IMAGE}:dev
+    command: python liconic_module/scripts/liconic_rest_node.py --port 2010 --device ${DEVICE} --resources_path ${RESOURCES_PATH}
+    privileged: true
+    env_file: .env
+    volumes:
+      - ./scripts:/home/app/liconic_module/scripts
```

```suggestion
      - ./scripts:/home/app/liconic_module/scripts
      - /dev:/dev
```

Need to mount in the device folder so we can actually communicate with it.